package.json

```diff
@@ -20,6 +20,7 @@
     "lint:check": "yarn lint:ts-logic && forge fmt --check",
     "lint:fix": "sort-package-json && forge fmt && yarn lint:ts-logic --fix",
     "lint:ts-logic": "eslint ./src/**/*.ts",
+    "smock-test": "node dist/run.js --contracts solidity --mocks solidity/test/smock --ignore solidity/test",
     "test": "forge build --skip test && cross-env mocha 'test/**/*.spec.ts' && forge test",
```

Comment:
What do you think about adding `yarn smock-test` to the `test` command, so that whoever runs `yarn test` locally will be able to execute the forge tests?